system/iptables_raw.py

```diff
@@ -4,6 +4,7 @@
 """
 (c) 2016, Strahinja Kustudic <strahinjak@nordeus.com>
 (c) 2016, Damir Markovic <damir@damirda.com>
+(c) 2016, Allen Sanabria <asanabria@linuxdynasty.org>
```

Comment:
I really appreciate the contribution, but I don't think that every contributor should be an author.